charts/common/templates/ingress.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.ingress.enabled -}}
+{{- if and .Values.ingress.enabled (not .Values.canary.enabled) -}}
```

Comment:
A couple of topics we spoke during our meeting, Disabling ingress in the chart by default can result outage during migration nginx>mesh test/failover, we might want to keep this one enabled to allow a planned migration path for helm chart and dns cutover. Also while keeping this one enable can result on putting traffic on canary

Reply:
@ofrivera so this is by design because canary only supports Cloud Armor for public ingress.